services/apps/alcs/src/alcs/application-decision/application-decision-outcome.entity.ts

```diff
@@ -0,0 +1,10 @@
+import { Column, Entity } from 'typeorm';
+import { BaseCodeEntity } from '../../common/entities/base.code.entity';
+
+@Entity()
+export class ApplicationDecisionOutcomeCode extends BaseCodeEntity {
+  @Column({
+    default: true,
+  })
+  isFirstDecision: boolean;
```

Comment:
So this one changed in scope of NOI and isFirst will stored on db level for both decision v1 and v2?

Reply:
I did not change this file other than renaming it and moving it.